build_tools/packaging/python/templates/rocm/src/rocm_sdk/_dist_info.py

```diff
@@ -234,6 +234,8 @@ def determine_target_family() -> str:
 LibraryEntry("hipfft", "libraries", "libhipfft.so*", "hipfft*.dll")
 LibraryEntry("hiprand", "libraries", "libhiprand.so*", "hiprand*.dll")
 LibraryEntry("hipsparse", "libraries", "libhipsparse.so*", "hipsparse*.dll")
+# TODO: Omit hipsparselt until windows builds are enabled
+# LibraryEntry("hipsparselt", "libraries", "libhipsparselt.so*", "hipsparselt*.dll")
```

Comment:
You could also make this Linux-only for the moment by omitting a pattern for the .dll
```suggestion
LibraryEntry("hipsparselt", "libraries", "libhipsparselt.so*", "")
```

Reply:
Done https://github.com/ROCm/TheRock/pull/1659/commits/590253e4535a200e730e35993dc8b6db1f43250b